.gitattributes

```diff
@@ -0,0 +1,2 @@
+rpmostree-cxxrs.{h,cxx} linguist-generated=true
+rust/cxx.h linguist-generated=true
```

Comment:
OK, this is cool. TIL.
Might be worth adding a comment explaining this though.

Reply:
This is pretty standard AFAIK.